src/bun.js/bindings/SQLClient.cpp

```diff
@@ -102,9 +115,24 @@ static JSC::JSValue toJS(JSC::VM& vm, JSC::JSGlobalObject* globalObject, DataCel
     case DataCellTag::Null:
         return jsNull();
         break;
+    case DataCellTag::Raw: {
+        Zig::GlobalObject* zigGlobal = jsCast<Zig::GlobalObject*>(globalObject);
+        auto* subclassStructure = zigGlobal->JSBufferSubclassStructure();
+        auto* uint8Array = JSC::JSUint8Array::createUninitialized(globalObject, subclassStructure, cell.value.raw.length);
+        if (UNLIKELY(uint8Array == nullptr)) {
+            return {};
+        }
+
+        if (cell.value.raw.length > 0) {
+            memcpy(uint8Array->vector(), reinterpret_cast<void*>(cell.value.raw.ptr), cell.value.raw.length);
+        }
+        return uint8Array;
+    }
     case DataCellTag::String: {
-        return jsString(vm, WTF::String(cell.value.string));
-        break;
+        if (cell.value.string) {
+            return jsString(vm, WTF::String(cell.value.string));
```

Comment:
Should the WTF::StringImpl in the cell be set to nullptr after this?
```suggestion
            return jsString(vm, WTF::String(cell.value.string));
```

Reply:
we need to deref it later so we still keepit